testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestHiveAndDeltaLakeRedirect.java

```diff
@@ -865,6 +847,26 @@ public void testHiveToDeltaPropertiesRedirect()
         }
     }
 
+    @Test(groups = {DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    public void testHiveToDeltaPartitionsRedirect()
+    {
+        String tableName = "test_redirect_to_delta_partitions_" + randomNameSuffix();
+
+        onTrino().executeQuery(createTableInDeltaConnector("default", tableName, true));
```

Comment:
Reason for creating this in Trino instead of Spark is because Spark seems to truncate string from `comment` when writing to transaction log.

When running the following in Spark:
```
CREATE TABLE tiny.nation
USING DELTA
PARTITIONED BY (regionkey)
 AS VALUES
      (CAST(0 AS bigint), 'ALGERIA', CAST(0 AS bigint), 'haggle. carefully final deposits detect slyly agai'),
      (CAST(1 AS bigint), 'ARGENTINA', CAST(1 AS bigint), 'al foxes promise slyly according to the regular accounts. bold requests alon'),
      (CAST(2 AS bigint), 'BRAZIL', CAST(1 AS bigint), 'y alongside of the pending deposits. carefully special packages are about the ironic forges. slyly special'),
      (CAST(3 AS bigint), 'CANADA', CAST(1 AS bigint), 'eas hang ironic, silent packages. slyly regular packages are furiously over the tithes. fluffily bold')
       AS data(nationkey, name, regionkey, comment);
```

The following transaction log `0000000000000000000.json` is created:
```
{"commitInfo":{"timestamp":1714195167896,"operation":"CREATE TABLE AS SELECT","operationParameters":{"isManaged":"true","description":null,"partitionBy":"[\"regionkey\"]","properties":"{}"},"isolationLevel":"Serializable","isBlindAppend":true,"operationMetrics":{"numFiles":"4","numOutputRows":"4","numOutputBytes":"6033"},"engineInfo":"Apache-Spark/3.5.0 Delta-Lake/3.1.0","txnId":"5948b862-92af-4fd8-abb9-b451c4cd7eda"}}
{"metaData":{"id":"b2884785-c03e-4785-b5eb-cb89f6adb0ea","format":{"provider":"parquet","options":{}},"schemaString":"{\"type\":\"struct\",\"fields\":[{\"name\":\"nationkey\",\"type\":\"long\",\"nullable\":true,\"metadata\":{}},{\"name\":\"name\",\"type\":\"string\",\"nullable\":true,\"metadata\":{}},{\"name\":\"regionkey\",\"type\":\"long\",\"nullable\":true,\"metadata\":{}},{\"name\":\"comment\",\"type\":\"string\",\"nullable\":true,\"metadata\":{}}]}","partitionColumns":["regionkey"],"configuration":{},"createdTime":1714195165727}}
{"protocol":{"minReaderVersion":1,"minWriterVersion":2}}
{"add":{"path":"regionkey=0/part-00000-9715f76d-7730-4dd0-ae0e-48cfba60fb37.c000.snappy.parquet","partitionValues":{"regionkey":"0"},"size":1322,"modificationTime":1714195167000,"dataChange":true,"stats":"{\"numRecords\":1,\"minValues\":{\"nationkey\":0,\"name\":\"ALGERIA\",\"comment\":\"haggle. carefully final deposits\"},\"maxValues\":{\"nationkey\":0,\"name\":\"ALGERIA\",\"comment\":\"haggle. carefully final deposits�\"},\"nullCount\":{\"nationkey\":0,\"name\":0,\"comment\":0}}"}}
{"add":{"path":"regionkey=1/part-00001-32f9315d-52f2-4e7c-a430-15839f80daf5.c000.snappy.parquet","partitionValues":{"regionkey":"1"},"size":1494,"modificationTime":1714195167000,"dataChange":true,"stats":"{\"numRecords\":1,\"minValues\":{\"nationkey\":1,\"name\":\"ARGENTINA\",\"comment\":\"al foxes promise slyly according\"},\"maxValues\":{\"nationkey\":1,\"name\":\"ARGENTINA\",\"comment\":\"al foxes promise slyly according�\"},\"nullCount\":{\"nationkey\":0,\"name\":0,\"comment\":0}}"}}
{"add":{"path":"regionkey=1/part-00002-00315589-a2f1-439e-bf15-006ac7beb3b4.c000.snappy.parquet","partitionValues":{"regionkey":"1"},"size":1624,"modificationTime":1714195167000,"dataChange":true,"stats":"{\"numRecords\":1,\"minValues\":{\"nationkey\":2,\"name\":\"BRAZIL\",\"comment\":\"y alongside of the pending depos\"},\"maxValues\":{\"nationkey\":2,\"name\":\"BRAZIL\",\"comment\":\"y alongside of the pending depos�\"},\"nullCount\":{\"nationkey\":0,\"name\":0,\"comment\":0}}"}}
{"add":{"path":"regionkey=1/part-00003-6c237370-1a44-40ef-9dd7-fe7101e93dd7.c000.snappy.parquet","partitionValues":{"regionkey":"1"},"size":1593,"modificationTime":1714195167000,"dataChange":true,"stats":"{\"numRecords\":1,\"minValues\":{\"nationkey\":3,\"name\":\"CANADA\",\"comment\":\"eas hang ironic, silent packages\"},\"maxValues\":{\"nationkey\":3,\"name\":\"CANADA\",\"comment\":\"eas hang ironic, silent packages�\"},\"nullCount\":{\"nationkey\":0,\"name\":0,\"comment\":0}}"}}
```

Reply:
FYI this is the same as `io.trino.plugin.deltalake.BaseDeltaLakeSharedMetastoreWithTableRedirectionsTest#testPartitionsTable`

Reply:
Should I get rid of this test then since it's similar to another test?

Also removed querying for `total_size` in expected for this test since the value is pretty transient, thus breaking the test. Removing prevent this test from randomly breaking in the future.

Reply:
This test doesn't depend on Spark or Databricks. I'm fine with removing it.

Reply:
👍